core/pom.xml

```diff
@@ -896,6 +896,26 @@
 
 	<build>
 		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>log4j2-plugin-processor</id>
+						<goals>
+							<goal>compile</goal>
+						</goals>
+						<phase>process-classes</phase>
+						<configuration>
+							<proc>only</proc>
+							<compileSourceRoots>${project.build.directory}/generated-sources/delombok</compileSourceRoots>
+							<annotationProcessors>
+								<annotationProcessor>org.apache.logging.log4j.core.config.plugins.processor.PluginProcessor</annotationProcessor>
+							</annotationProcessors>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
```

Comment:
Waarom is dit eigenlijk nodig?

Reply:
Zodat Log4j2 de plugins die in het logging package zitten scan't en gebruikt.
Zonder werkt het prima runtime, maar niet voor tests.
En Log4j heeft er voor gekozen om de package functionaliteit zoals die nu bestaat in versie 3 verwijderd wordt, dus er is een alternatief nodig. Helaas ben ik er nog niet helemaal uit hoe dat er dat uit komt te zien, maar dit is denk ik een eerste stap.

Reply:
Dit is dus alleen nodig voor de tests. 👍